pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py

```diff
@@ -367,6 +368,10 @@ def __init__(self, name="Doyle-Fuller-Newman half cell model", options=None):
             + pot * vdrop_cell,
             "Terminal voltage": voltage,
             "Terminal voltage [V]": phi_s_w_ref + param.U_n_ref + pot * voltage,
+            "Instantaneous power [W.m-2]": i_cell * i_typ * (phi_s_w_ref + param.U_n_ref + pot * voltage),
+            "Pore-wall flux [mol.m-2.s-1]": j_w,
+            "Ratio of electrolyte diffusion to discharge time": param.C_e,
+            "Ratio of solid diffusion to discharge time": param.C_p,
```

Comment:
you can print parameters to a file like this https://github.com/pybamm-team/PyBaMM/blob/d62c862d00845272e8111f91f9541f0e0c484dde/tests/unit/test_parameters/test_lithium_ion_parameters.py#L12-L15

Reply:
Thanks.

Reply:
So you don't need to add these to the list of variables